### PR TITLE
add ppc patches for 7.3.2, will be fixed in next release

### PR DESCRIPTION
**Pull request by mattip**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattip** [comment]
@conda-forge-admin, please rerender

**Comment by mattip** [comment]
it seems the ppc build is not run

**Comment by isuruf** [comment]
@mattip, ppc64le build takes longer than the 50 minutes travis-ci lets us, so I run it locally. When the PR is ready, I'll do that.

**Comment by mattip** [comment]
whoops. Good catch. Fixed, thanks.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,9 +20,13 @@ source:
       - patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
       - patches/0013-Disable-new-dtags-in-unixccompiler.py.patch
       - patches/0020-Use-ranlib-from-env-if-env-variable-is-set.patch
+      - patches/0020-ppc-jit-fix
+      - patches/0021-ooc-test-fix1
```
Comment (isuruf) [inline recipe/meta.yaml:24]:
```suggestion
      - patches/0021-ppc-test-fix1
```

Comment (mattip) [reply recipe/meta.yaml]: aaargh. Thanks again.

**Comment by mattip** [comment]
Fixes #28

**Comment by isuruf** [comment]
Thanks

**Comment by mattip** [comment]
Is this live? I see someone tried to re-render numpy in conda-forge/numpy-feedstock#204 and [ppc pypy failed](https://github.com/conda-forge/numpy-feedstock/pull/204/checks?check_run_id=1232209529).

**Pull request merged**